db/migrate/20170531194154_add_multiple_redirect_uris.rb

```diff
@@ -0,0 +1,15 @@
+class AddMultipleRedirectUris < ActiveRecord::Migration
+  def up
+    add_column :service_providers, :redirect_uris, :json
```

Comment:
I wouldn't consider choice of column types a style decision. Wouldn't it make sense to choose the right column type for the job?